src/core-app-plane/core-app-plane.ts

```diff
@@ -103,49 +33,10 @@ export class CoreApplicationPlane extends Construct {
   constructor(scope: Construct, id: string, props: CoreApplicationPlaneProps) {
     super(scope, id);
     addTemplateTag(this, 'CoreApplicationPlane');
-    this.eventManager = props.eventManager ?? new EventManager(this, 'EventManager');
-    const eventBus = EventBus.fromEventBusArn(this, 'EventBus', this.eventManager.busArn);
-
-    props.jobRunnerPropsList?.forEach((jobRunnerProps) => {
-      // Only BashJobOrchestrator requires differentiating between
-      // strings and JSON variables pulled from the incoming event.
-      let envVarsFromIncomingEvent: string[] = [];
-      if (jobRunnerProps.environmentStringVariablesFromIncomingEvent) {
-        envVarsFromIncomingEvent.concat(jobRunnerProps.environmentStringVariablesFromIncomingEvent);
-      }
-
-      if (jobRunnerProps.environmentJSONVariablesFromIncomingEvent) {
-        envVarsFromIncomingEvent.concat(jobRunnerProps.environmentJSONVariablesFromIncomingEvent);
-      }
-
-      let job = new BashJobRunner(this, jobRunnerProps.name, {
-        name: jobRunnerProps.name,
-        permissions: jobRunnerProps.permissions,
-        script: jobRunnerProps.script,
-        postScript: jobRunnerProps.postScript,
-        environmentVariablesFromIncomingEvent: envVarsFromIncomingEvent,
-        environmentVariablesToOutgoingEvent: jobRunnerProps.environmentVariablesToOutgoingEvent,
-        scriptEnvironmentVariables: jobRunnerProps.scriptEnvironmentVariables,
-      });
-
-      let jobOrchestrator = new BashJobOrchestrator(this, `${jobRunnerProps.name}-orchestrator`, {
-        targetEventBus: eventBus,
-        detailType: jobRunnerProps.outgoingEvent,
-        eventSource: this.eventManager.supportedEvents[jobRunnerProps.outgoingEvent],
-        environmentVariablesToOutgoingEvent: jobRunnerProps.environmentVariablesToOutgoingEvent,
-        environmentStringVariablesFromIncomingEvent:
-          jobRunnerProps.environmentStringVariablesFromIncomingEvent,
-        environmentJSONVariablesFromIncomingEvent:
-          jobRunnerProps.environmentJSONVariablesFromIncomingEvent,
-        bashJobRunner: job,
-        tenantIdentifierKeyInIncomingEvent: jobRunnerProps.tenantIdentifierKeyInIncomingEvent,
-      });
+    this.eventManager = props.eventManager;
 
-      this.eventManager.addTargetToEvent(
-        this,
-        jobRunnerProps.incomingEvent,
-        jobOrchestrator.eventTarget
-      );
+    props.jobRunnersList?.forEach((jobRunner) => {
+      this.eventManager.addTargetToEvent(this, jobRunner.incomingEvent, jobRunner.eventTarget);
```

Comment:
This is really the only thing that the `CoreApplicationPlane` construct is doing now. I think we should just get rid of it and pass the eventManager into the BashJob runner so that it's able to hook into the eventManager on it's own.

Reply:
I did debated on this a bit and thought of keeping it atleast for couple more iterations because
 1/ This class can be like a controller which can switch individual objects  2/ Maybe for future changes we might need it.